web/index.php

```diff
@@ -2,14 +2,81 @@
 
 define('APP_PATH', realpath(__DIR__ . '/..') . DIRECTORY_SEPARATOR);
 
+use josegonzalez\Dotenv\Loader as EnvLoader;
+use Facebook\Facebook;
+use \Github\Client as GithubClient;
+use TwitterAPIExchange;
+use \HieuLe\WordpressXmlrpcClient\WordpressClient;
+
 require APP_PATH . 'vendor/autoload.php';
 
+$services = 'Wheniwork\Feedback\Service';
+
+// Initialize Injector
 $injector = new Auryn\Injector;
 $injector->define('Predis\Client', []);
+$injector->define(EnvLoader::class, [':filepaths' => __DIR__ . '/../.env']);
+$injector->prepare(EnvLoader::class, function(EnvLoader $loader) {
+    $loader->parse()->putenv(true);
+});
+$injector->delegate("$services\AppStoreService", function() {
+    return new Wheniwork\Feedback\Service\AppStoreService(
+        getenv('WIW_IOS_APP_ID')
+    );
+});
+$injector->delegate("$services\BlogService", function() {
+    $client = new WordpressClient(
+        "http://wheniwork.com/blog/xmlrpc.php",
+        getenv('WP_USER'),
+        getenv('WP_PASSWORD')
+    );
+    return new Wheniwork\Feedback\Service\BlogService($client);
+});
+$injector->delegate("$services\FacebookService", function() {
+    $fb = new Facebook([
+        'app_id' => getenv('FB_APP_ID'),
+        'app_secret' => getenv('FB_APP_SECRET'),
+        'default_graph_version' => 'v2.4'
+    ]);
+    $service = new Wheniwork\Feedback\Service\FacebookService($fb, getenv('FB_PAGE_ID'));
```

Comment:
Technically, the service factories should also be pulled from the injector to allow overloading. But since this is all happening in the front controller, it's okay to do it this way.